### ISSUE TITLE
How to know the loss of every epoch?

### ISSUE THREAD

**Issue by nomancseku** [tensorflow/models#9889]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [x] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [x] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/official/...

## 3. Steps to reproduce

Steps to reproduce the behavior.

## 5. Additional context

Include any logs that would be helpful to diagnose the problem.

## 6. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- Mobile device name if the issue happens on a mobile device:
- TensorFlow installed from (source or binary):
- TensorFlow version (use command below):
- Python version:
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Comment by saikumarchalla** [comment]
@nomancseku Looks the issue is more related to TF core Repo. Also, Please fill the issue template and let us know the top level directory of the model you are using.

**Comment by saberkun** [comment]
Some callbacks like keras TensorBoard https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard can report that. Also, keras default will report epoch loss?
If you write your own code, you can report the loss of the entire epoch loop.
